config/initializers/exception_notification.rb

```diff
@@ -22,7 +22,7 @@
   #
 
   config.add_notifier :slack, {
-    webhook_url: Settings.slack_webhook,
+    webhook_url: Settings.slack_webhook.match(%r{^https://}) && Settings.slack_webhook,
```

Comment:
Nit: This should be equivalent:

```suggestion
    webhook_url: Settings.slack_webhook[%r{^https://.*$}],
```

Reply:
Nice!